Generator.API/Controllers/GeneratorController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System.Threading.Tasks;
+using Generator.Application.Commands;
+using Generator.Application.Dtos;
+using Generator.Application.Queries;
+using MediatR;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Generator.API.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class GeneratorController : ControllerBase
```

Comment:
controller moze byc pictures to wtedy w tych action nie musialbys powtarzac tego samego